test/test_distributed.py

```diff
@@ -1013,6 +1014,29 @@ def test_all_reduce_full_group_max(self):
             group, group_id, rank, dist.ReduceOp.MAX, -1, 10, 10
         )
 
+    # SPARSE ALL REDUCE
+    def _test_sparse_all_reduce_sum(self, fn):
+        group, group_id, rank = self._init_global_test()
+
+        tests = simple_sparse_reduce_tests(
+            rank,
+            dist.get_world_size(),
+            num_inputs=1)
+        for (inputs, outputs) in tests:
+            tensors = [fn(input) for input in inputs]
+            dist.all_reduce(tensors[0], dist.ReduceOp.SUM, group_id)
```

Comment:
I might miss sth here, why this is only testing `tensors[0]`? What about other tensors?

Reply:
because dist_all_reduce() only accepts one tensor right now? you can see in the test I intentionally only tested for inputs with size 1.